lib/BackgroundJobs/IndexerJob.php

```diff
@@ -265,7 +265,11 @@ protected function index(array $files): void {
 		$emptyInvalidSources = array_diff($allSourceIds, $loadedSources);
 		if (count($emptyInvalidSources) > 0) {
 			$this->logger->info('Invalid or empty sources that were not indexed (n=' . count($emptyInvalidSources) . ')', ['sourceIds' => $emptyInvalidSources]);
-			$this->logger->info('Invalid or empty files that were not indexed: ' . json_encode(array_map(fn ($sourceId) => $this->rootFolder->getFirstNodeById(intval(explode(' ', $sourceId)[1]))->getPath(), $emptyInvalidSources)));
+			try {
+				$this->logger->info('Invalid or empty files that were not indexed: ' . json_encode(array_map(fn ($sourceId) => $this->rootFolder->getFirstNodeById(intval(explode(' ', $sourceId)[1]))->getPath(), $emptyInvalidSources)));
+			} catch (\Exception $e) {
+				$this->logger->warning('Could not get invalid or empty files that were not indexed', ['exception' => $e]);
```

Comment:
`$emptyInvalidSources` can be printed without beautification still

Reply:
that's the line above :)

Reply:
ahh hehe